go/worker/compute/merge/committee/node.go

```diff
@@ -76,6 +77,12 @@ var (
 	infiniteTimeout = time.Duration(math.MaxInt64)
 )
 
+// Parameters for retrying local submission calls.
+const (
+	retryInterval = 1 * time.Second
+	maxRetries    = 15
```

Comment:
This should be [the same as in P2P](https://github.com/oasisprotocol/oasis-core/blob/cfc8b1a0d12e0e1fe425d987b6149643b844a015/go/worker/common/p2p/p2p.go#L194) which just uses `NewExponentialBackOff`. No need to limit the retries as the round context will be cancelled in case the round advances.